swift/ql/test/library-tests/ast/PrintAst.expected

```diff
@@ -6651,62 +6643,24 @@ patterns.swift:
 #  180|           getPattern(): [AnyPattern] _
 #  181|         getBody(): [BraceStmt] { ... }
 #  181|           getElement(0): [TupleExpr] (...)
-#   83| [ConcreteVarDecl] a
-#   83|     Type = (Int)
-#   85| [ConcreteVarDecl] a
-#   85|     Type = Int
-#   85| [ConcreteVarDecl] b
-#   85|     Type = Int
-#   88| [ConcreteVarDecl] a
-#   88|     Type = Int
 #   92| [ConcreteVarDecl] x
 #   92|     Type = (Int)
 #   95| [ConcreteVarDecl] x
 #   95|     Type = Int
```

Comment:
still quite a number of orphaned (or maybe duplicate?) var decls here, eh?

Reply:
Looks like it, yeah